js/projects.js

```diff
@@ -0,0 +1,34 @@
+console.log('it works');
+var projects = [];
+
+function Project(opts){
+  this.title = opts.ttile;
+  this.developer = opts.developer;
+  this.date = opts.date;
+  this.projectURL = opts.projectURL;
+  this.projectDescription = opts.projectDescription;
+}
+
+Project.prototype.toHtml = function(){
+  var $newProject = $('projects.project-location').clone();
```

Comment:
.project-location exists, but where's the element projects? Do you mean article?